packages/flutter_tools/lib/src/commands/test.dart

```diff
@@ -694,7 +694,7 @@ class TestCommand extends FlutterCommand with DeviceBasedDevelopmentArtifacts {
   }) async {
     final AssetBundle assetBundle = AssetBundleFactory.instance.createBundle();
     final int build = await assetBundle.build(
-      packagesPath: '.packages',
+      packageConfigPath: '.dart_tool/package_config.json',
```

Comment:
why do we hard-code here? do we not support workspaces in tests?

Reply:
I guess just to make existing tests continue working.
If we want to make many workspace tests it could totally be parametrized.

Reply:
Done